lib/Jsrt/ChakraCore.h

```diff
@@ -1035,5 +1035,33 @@ CHAKRA_API
     JsSetHostPromiseRejectionTracker(
         _In_ JsHostPromiseRejectionTrackerCallback promiseRejectionTrackerCallback, 
         _In_opt_ void *callbackState);
+
+/// <summary>
+///     Determines if a provided object is a JavscriptProxy Object and
+///     provides references to a Proxy's target and handler.
+/// </summary>
+/// <remarks>
+///     Requires an active script context.
+///     If object is not a Proxy object the target and handler parameters are not touched.
+///     If nullptr is supplied for target or handler the function returns after
+///     setting the isProxy value.
+///     If the object is a revoked Proxy target and handler are made to point to nullptr.
```

Comment:
I think we should set the target and handler to `JS_INVALID_REFERENCE` instead of nullptr and have this reflected in the doc here.